notebooks/text_classification/text_classification_classical_text_explainer.ipynb

```diff
@@ -351,7 +358,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.6.8"
```

Comment:
nit: same for python version in kernel (and all other metadata) - make sure it is consistent for all notebooks